### ISSUE TITLE
Regarding the discrepancy in experimental results of cut_roasted_beef in N3DV dataset.

### ISSUE THREAD

**Issue by LabubuJ** [juno181/Ex4DGS#10]
Thank you for your excellent work. I encountered some issues during my experiments.
The metric of cut_roasted_beef I got is {
 "SSIM": 0.9544036984443665,
 "SKSSIM": 0.9497656226158142,
 "SKSSIM2": 0.9747065305709839,
 "PSNR": 31.879066467285156,
 "LPIPS": 0.047466572374105453,
 "LPIPSVGG": 0.13830572366714478,
 "times": 0.01961996965110302
}
There is a significant discrepancy from the PSNR of 33.73 provided in the paper.
I used the code and commands you provided, and I would like to know how to obtain normal results.

**Comment by LabubuJ** [comment]
The training configuration is as follows.        
{
            "name": "Python train cut_roasted_beef",
            "type": "debugpy",
            "request": "launch",
            "program": "./train.py",
            "console": "integratedTerminal",
            "args": [
                "-s", "/home/disk1/jh/E-D3DGS/dataset/cut_roasted_beef_ed3dgs",
                "--config", "/home/disk1/jh/E-D3DGS/arguments/dynerf/cut_roasted_beef.py",
                "--model_path", "/home/disk1/jh/E-D3DGS/output/cut_roasted_beef",
                "--expname", "dynerf/cut_roasted_beef",
                "-r", "2",
            ],
            "python": "/home/disk1/jh/conda_envs/ed3dgs/bin/python"
        }

**Comment by juno181** [comment]
Thanks for your experiment. This appears to be the same issue as #3. I am suspecting that the problem is that COLMAP is not producing the same results. I am currently revising the proprocessing and will update the code.

**Comment by LabubuJ** [comment]
Thanks for your response！ Looking forward to your update.发自我的手机-------- 原始邮件 --------发件人： Juno ***@***.***>日期： 2024年11月24日周日 晚上9:33收件人： juno181/Ex4DGS ***@***.***>抄送： JiaoChenGuang-JiaoHan ***@***.***>, Author ***@***.***>主    题： Re: [juno181/Ex4DGS] Regarding the discrepancy in experimental results of cut_roasted_beef in N3DV dataset. (Issue #10)
Thanks for your experiment. This appears to be the same issue as #3. I am suspecting that the problem is that COLMAP is not producing the same results. I am currently revising the proprocessing and will update the code.

**Comment by YJ-LIAO** [comment]
Did you completely revise the processing code? I saw you submit a new pre_n3d_colmap.py, but I also have a bad result in coffee_martini. The metric of coffee_martini I got is PSNR: 28.13.

**Comment by YJ-LIAO** [comment]
> Thanks for your experiment. This appears to be the same issue as [#3](https://github.com/juno181/Ex4DGS/issues/3). I am suspecting that the problem is that COLMAP is not producing the same results. I am currently revising the proprocessing and will update the code.

Did you completely revise the processing code? I saw you submit a new pre_n3d_colmap.py, but I also have a bad result in coffee_martini. The metric of coffee_martini I got is PSNR: 28.13.

**Comment by juno181** [comment]
The new version of pre_n3d_colmap.py was submitted primarily to address issues related to resolution. However, the parts related to COLMAP densification are still being worked on and haven't been completely revised yet.

**Comment by YJ-LIAO** [comment]
> The new version of pre_n3d_colmap.py was submitted primarily to address issues related to resolution. However, the parts related to COLMAP densification are still being worked on and haven't been completely revised yet.

I found that the reconstruction evaluation results for dense inputs are much closer to those for sparse inputs in the paper, is this normal?

**Comment by juno181** [comment]
That's correct. This is because a dense input reduces the randomness of densification, which makes the result more stable. I am improving this by using COLMAP densification.

**Comment by TCQian** [comment]
> That's correct. This is because a dense input reduces the randomness of densification, which makes the result more stable. I am improving this by using COLMAP densification.

I am facing the same issue here. Is the COLMAP densification ready yet?
